src/App.tsx

```diff
@@ -43,6 +44,12 @@ function App() {
             path="/exercises/create"
             element={<ProtectedRoute element={<CreateExercisePage />} />}
           />
+
+            <Route
+            path="/category/create"
```

Comment:
Let's be uniform (everything is in plurar)

```suggestion
            path="/categories/create"
```